src/uu/arch/arch.md

```diff
@@ -0,0 +1,14 @@
+# arch
+
+```
+arch
+```
+
+```
```

Comment:
The code fences for about and after help will lead to weird formatting. (The first fences for the usage are correct though)